File: MORM.Java/src/MORM/Java/Models/Venda/Terminal.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "TERMINAL")
public class Terminal extends CollectionItem
{
    @CampoAttribute(Campo = "ID_TERMINAL", Tipo = CampoTipo.Key)
    public Integer Id_Terminal;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Req)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Req)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "CD_TERMINAL", Tipo = CampoTipo.Req)
    public Integer Cd_Terminal;
    @CampoAttribute(Campo = "DS_TERMINAL", Tipo = CampoTipo.Req)
    public String Ds_Terminal;
}

File: MORM.CSharp/MORM.Dominio/Models/Venda/Terminal.cs
using MORM.Util.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("TERMINAL")]
    public class Terminal : CollectionItem
    {
        [Campo("ID_TERMINAL", CampoTipo.Key)]
        public int Id_Terminal { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Req)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Req)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("CD_TERMINAL", CampoTipo.Req)]
        public int Cd_Terminal { get; set; }
        [Campo("DS_TERMINAL", CampoTipo.Req)]
        public string Ds_Terminal { get; set; }
    }
}